app/src/main/java/com/fluffybros/tujefe/ui/home/HomeFragment.kt

```diff
@@ -4,11 +4,11 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.TextView
 import androidx.fragment.app.Fragment
-import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
 import com.fluffybros.tujefe.R
+import kotlinx.android.synthetic.main.fragment_home.*
```

Comment:
kotlin synthetic views are no longer best practices - we should use [view binding](https://developer.android.com/topic/libraries/view-binding) instead